src/xdocs/checks/misc/finalparameters.xml

```diff
@@ -144,6 +152,57 @@ public class Example3 {
   public void methodOne(final int x) { }
   public void methodTwo(int x) { }
   public static void main(String[] args) { } // violation, 'args should be final'
+}
+        </source>
+        <p id="Example4-config">
+          To configure the check to enforce final parameters for catch and
+          for-each blocks while ignoring unnamed parameters.
+          Since unnamed parameters cannot be read from or written to,
+          marking them as final is unnecessary:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;FinalParameters&quot;&gt;
+      &lt;property name=&quot;tokens&quot; value=&quot;FOR_EACH_CLAUSE, LITERAL_CATCH&quot;/&gt;
+      &lt;property name=&quot;ignoreUnnamedParameters&quot; value=&quot;true&quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p id="Example4-code">
+          Example:
+        </p>
+        <source>
+public class Example4 {
+
+  void testCatchParameters() {
+    try {
+      int x = 1 / 0;
+    }
+    catch (Exception e) { // violation, 'Parameter e should be final'
+      System.out.println(e);
+    }
+    try {
+      int x = 1 / 0;
+    }
+    catch (Exception _) { // ok, unnamed catch parameter, it is implicitly final
```

Comment:
Add another example with a `final` unnamed and just say it is ok to have the final declared.

Reply:
done